setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name="OpenFisca-France-Local",
-    version="5.1.0",
+    version="5.1.1",
```

Comment:
Ici on ajoute une Variable au Tax and Benefit system (on ajoute une aide) donc on ajoute une fonctionalité, c'est un changement mineur et nom un patch.
```suggestion
    version="5.2.0",
```